scss/book-list-detail.scss

```diff
@@ -0,0 +1,85 @@
+.body{
+  font-family: 'Roboto', sans-serif;
+}
+
+.master-container{
+  display: flex;
+  height: 768px;
+  width: 1366px;
+  background-color: $wild-sand;
+
+  .navbar {
+    position: fixed;
+    display: flex;
+    top: 0;
+    left: 0;
+    border-top: 6px solid $cerulean; 
+    height: 70px;
+    width: 100%;
+    max-width: 1366px;
+    background-color: $white;
+    box-shadow: 0 4px 10px 0 $blackRGBA;
+    align-items: center;
+    align-content: center;
+    justify-content: space-around;
+    
+    .wolox-logo {
+      height: 70px;
+      width: 212px;
+    }
+
+    .logout {
+      color: $black;
+      font-size: 18px;
+      line-height: 21px;
+      margin-left: 150px;
```

Comment:
Is this margin necessary?

Reply:
Not really, it makes it look more like the image i had as reference but really isnt necessary